tools/config.py

```diff
@@ -477,10 +489,11 @@ def has_regions(self):
     @property
     def regions(self):
         """Generate a list of regions from the config"""
-        if not self.target.bootloader_supported:
+        if not (self.target.bootloader_supported and \
+                self.target.flash_erase_size and \
+                self.target.vector_table_size):
```

Comment:
@sg- Do we want to put the flash erase size and the vector table size into `targets.json`?